app/helpers/document_filter_helper.rb

```diff
@@ -11,6 +11,11 @@ def organisation_filter_options(selected_organisations = [])
     unsorted_grouped_options_for_select(filter_options.for(:organisations).grouped, selected_values)
   end
 
+  def people_filter_options(selected_person = [])
```

Comment:
If this is an array (admittedly one that will only have one thing in it) it should probably be plural `selected_people`.